Disable epoll on Linux since our libevent is new enough to have
a fixed select/poll backend.
---
 osdep-linux.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/osdep-linux.c
+++ b/osdep-linux.c
@@ -71,6 +71,6 @@
 	 * poll or select, causing hangs.
 	 * 
 	 */
-	/* setenv("EVENT_NOEPOLL", "1", 1); */
+	setenv("EVENT_NOEPOLL", "1", 1);
 	return (event_init());
 }
